models/reviews_models.js

```diff
@@ -62,7 +67,7 @@ exports.updateReviewById = (body, params) => {
       "comment_count",
     ];
     const correctOrder = ["desc", "asc"];
-  
+    const queryValues = []
   
   let queryString = `
   SELECT
```

Comment:
You could refactor this to just make it select everything from Reviews if that is what you wanted